modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -670,6 +670,7 @@
                               :percentile-aggregations  true
                               :metadata/key-constraints false
                               :identifiers-with-spaces  true
+                              :cast                     true
```

Comment:
Are we confident this doesn't have unintended consequences? I don't know what else this feature might control.

Reply:
That's a good question. I don't really know how it works.

There are three expressions that require `:cast`: `text()`, `date()`, and `integer()`. Does it mean that if we put `:cast true` here, that the frontend expects all three?

@alexander-yakushev